### PR TITLE
Update redirect-known-folders.md

### PR DESCRIPTION
**Pull request by JeremyEsposito**
Please review my update. We are seeing many support cases where KFM is failing due to customers having custom Windows Known Folder GPO's for Documents, Pictures, or Desktop folders.

### CODE REVIEW

File: OneDrive/redirect-known-folders.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ There are two primary advantages of moving or redirecting Windows known folders
 
 For these reasons, we recommend moving or redirecting known folders to OneDrive if you're an enterprise or large organization. [See all our recommendations for configuring the sync app](ideal-state-configuration.md). Small or medium businesses may also find this useful, but keep in mind you'll need some experience configuring policies. For info about the end-user experience, see [Protect your files by saving them to OneDrive](https://support.office.com/article/d61a7930-a6fb-4b95-b28a-6552e77c3057). 
 
+> [!NOTE]
+  > Before enabling KFM, validate you do not have any existing 'Windows Known Folder' GPO's configurationed for Documents, Pictures, or Desktop folders.
+  > KFM will not deploy correctly if Documents, Pictures, or Desktop folders are pointing to their non-default location. 
+  > Also, Documents, Pictures, or Desktop folders must have their 'Location' setting to 'Unconfigured'.
```
Comment (JohanFreelancer9) [inline OneDrive/redirect-known-folders.md:44]:
```suggestion
> Before enabling Known Folder Move, validate that you do not have any existing _Windows Known Folder_ GPOs configured for Documents, Pictures, or Desktop folders.
>
> Known Folder Move will not deploy correctly if Documents, Pictures, or Desktop folders are pointing to their non-default location.
>
> Documents, Pictures, and Desktop folders must also have their **Location** set to **Unconfigured**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JeremyEsposito, kindly commit the suggested changes. Many thanks.

**Comment by JeremyEsposito** [comment]
Hi,
Please let me know what I need to do.
Sorry, I'm not familiar with these processes.

Let me know if you have any questions.
Thank you,
Jeremy Esposito
Sr. Support Escalation Engineer
O365 SharePoint - OneDrive for Business
Customer Service & Support
http://support.microsoft.com/<https://nam06.safelinks.protection.outlook.com/?url=http%3A%2F%2Fsupport.microsoft.com%2F&data=01%7C01%7Cjeesposi%40microsoft.com%7Cc562eb1ac3894f1008ef08d6d7cafed9%7C72f988bf86f141af91ab2d7cd011db47%7C1&sdata=yoSGBDaTsB%2B7xt723NGDgWpoal0YcKiPaFFT20NjATE%3D&reserved=0>
Phone:   +1 781 398 4724
Hours:    9:00AM - 5:30PM EST
***@***.******@***.***>
5 Wayside, Burlington, MA 01803
Manager: Nabil Mikkawi
***@***.******@***.***>

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mkbond007** [comment]
I'm currently working on an update for this article, and it will include these suggested changes.

**Pull request closed without merging**